source/boot_security.c

```diff
@@ -1,11 +1,21 @@
 #include "boot_security.h"
+#include "aes.h"
+#include "hal_log.h"
+#define EXAMPLE_AES_KEY_LEN_IN_BITS		256
+#define EXAMPLE_AESKEY_LEN_IN_BYTES		(EXAMPLE_AES_KEY_LEN_IN_BITS / 8)
```

Comment:
From now, we are not developing an example so please get rid of EXAMPLE_ prefix from the above defines. 

Also, let us not use AES definition here, instead, let's use "DEC" prefix for decryption.

#define DECRYPTION_KEY_LEN_IN_BITS                256
#define DECRYPTION_KEY_LEN_IN_BYTES             (DECRYPTION_KEY_LEN_IN_BITS / 8)

Reply:
Once you have done the above fix, please add a comment about the key length and why we use this length.

The comment could be like below
```

/*
 * Symmetric Key Length for Decryption
 * We use AES-256 for encryption/decryption
 */
#define DECRYPTION_KEY_LEN_IN_BITS 256
```